src/components/hero/Hero.tsx

```diff
@@ -15,7 +15,9 @@ const Hero: FunctionComponent = () => {
           <SkillsGroup title="Soft Skills" skills={softSkills} />
         </Column>
         <Column size={6} center>
-          <span>Foto</span>
+          <span>
+            <img className="header-avatar" src="./../img/foto.png" />

```

Comment:
Let's add all static images to `/public/static/media` and path for image will be `/static/media/[past name of your file here]`.